.github/workflows/triage_handle_new_comments.yml

```diff
@@ -0,0 +1,33 @@
+name: 'Handle Issue/PR Comment Workflow'
+on:
+  # makes this workflow reusable
+  workflow_call:
+    secrets:
+      ADD_TO_TRIAGE_BOARD_TOKEN:
+        required: true
+
+  issue_comment:
+    types: [created]
+
+jobs:
+  handle-comment-scenarios:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          repository: 'cypress-io/release-automations'
+          ref: 'master'
+          ssh-key: ${{ secrets.WORKFLOW_DEPLOY_KEY }} 
+      - name: Set up Node.js
```

Comment:
What's the default version of node on the runner?

Reply:
They switch when a new version becomes LTS.  I believe it is currently 16 but will likely switch to 18 sometime in the future.

Reply:
The default Node.js on all GitHub runners is currently `18.15.0` and probably there is no need to select Node.js 16 explicitly here.

(I know that this PR is closed! I'm just looking at it because it has broken other workflows. That has nothing to do with the Node.js version however.)